cmd/ui/v1alpha2/Dockerfile

```diff
@@ -2,7 +2,12 @@ FROM golang:alpine AS build-env
 # The GOPATH in the image is /go.
```

Comment:
Can you rebase with new Dockerfile for the Katib UI in v1alpha2, please ?

Reply:
OK. I will commit the v2 docker files. Thanks.

Reply:
OK.